src/components/tasks/TaskList.vue

```diff
@@ -1,6 +1,7 @@
 <template>
```

Comment:
I noticed that if the user adds too many items, the task list container expands to the bottom and the page starts to scroll. Maybe it would be better to make the task list container size static and add scrolling to the list container itself, in case there are a lot of tasks